src/styles/main.scss

```diff
@@ -1,3 +1,182 @@
-body {
+
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+* {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  max-width: 100vw;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.link_logo {
+  margin-top: 4px;
+}
+
+.nav {
+  &_list {
+    max-width: 1200px;
+    display: flex;
+    height: 60px;
+    padding: 0;
+  }
+
+  &_item {
+    margin-right: 20px;
+    list-style: none;
+    &:last-of-type {
+      margin-right: 0;
+    }
+    
+  }
+  &_link {
+    display: flex;
+    align-items: center;
+    text-decoration: none;
+    height: 60px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 14px;
+    color: #060b35;
+  }
+}
+
+.is-active {
+  position: relative;
+  display: flex;
+  align-items: center;
+  text-decoration: none;
+  height: 60px;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 14px;
+  &:after {
+    display: block;
+    position: absolute;
+    bottom: 0;
+    content: "";
+    border-radius: 2px;
+    height: 4px;
+    background-color: #00acdc;
+    width: 100%;
+    margin-left: 0;
+  }
+}
+
+.is-active,
+.nav_link:hover {
+  color: #00acdc;
+}
+
+.container {
+  padding-top: 50px;
+  margin: auto;
+  box-sizing: border-box;
+  max-width: 1200px;
+  min-width: 1024px;
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+}
+
+.product {
+  margin: 0 23px 47px;
+  box-sizing: border-box;
+  height: 409px;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  padding: 0 16px;
```

Comment:
It is forbidden to set sizes and margins for blocks, but for block elements are allowed. The block should stretch to accommodate its children